e2e/link_prediction_test/test_dataset_empty/test.yml

```diff
@@ -0,0 +1,7 @@
+# Completely empty dataset. We expect it to fail by throwing exception.
+
+query: >
+  CALL link_prediction.train()
+  YIELD training_results, validation_results
+  RETURN training_results, validation_results;
+exception: "Empty dataset. "
```

Comment:
Along with other checks, you could include the test for predicting edges. This can be achieved using stateful tests.